docs/source/model_doc/mbart.rst

```diff
@@ -76,6 +76,65 @@ the sequences for sequence-to-sequence fine-tuning.
     assert translation == "Şeful ONU declară că nu există o soluţie militară în Siria"
 
 
+MBart-50
+-----------------------------------------------------------------------------------------------------------------------
+
+mBART-50 is created using the original mBART model and extended to add extra 25 languages to support multilingual
+machine translation models of 50 languages. It was introduced in the `Multilingual Translation with Extensible
+Multilingual Pretraining and Finetuning <https://arxiv.org/abs/2008.00401>` paper by Yuqing Tang, Chau Tran, Xian Li,
+Peng-Jen Chen, Naman Goyal, Vishrav Chaudhary, Jiatao Gu, Angela Fan.
+
+According to the abstract
+
+*Multilingual translation models can be created through multilingual finetuning. Instead of finetuning on one
+direction, a pretrained model is finetuned on many directions at the same time. It demonstrate that pretrained models
+can be extended to incorporate additional languages without loss of performance. Multilingual finetuning improves on
+average 1 BLEU over the strongest baselines (being either multilingual from scratch or bilingual finetuning) while
+improving 9.3 BLEU on average over bilingual baselines from scratch.*
+
+
+Training
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The text format for mBART-50 is slightly different from mBART-25. For mBART-50 the language id token is used as prefix
+for both source and target text i.e the text format is :obj:`[lang_code] X [eos]` for both source and target text,
```

Comment:
what is `X`? Maybe just add a `with X being the source or target text respectively